evm/gas-service.js

```diff
@@ -304,24 +314,42 @@ async function processCommand(config, chain, options) {
             }
 
```

Comment:
we can run the script in `--offline` mode in the cron. This will skip connecting to the config RPC. A dummy private key can be set in the cron